src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6868,8 +6868,8 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="INF_TooManyBoundLambdas_Title" xml:space="preserve">
     <value>Compiling requires binding the lambda expression many times. Consider declaring the lambda expression with explicit parameter types, or if the containing method call is generic, consider using explicit type arguments.</value>
   </data>
-  <data name="INF_IdentifierConflictWithContextualKeyword" xml:space="preserve">
-    <value>'{0}' is a contextual keyword in property accessors starting in language version {1}. Use '@{0}' instead.</value>
+  <data name="WRN_FieldIsAmbiguous" xml:space="preserve">
+    <value>'{0}' binds to the synthesized backing field for the property in language version {1}. Use '@{0}' to bind to the existing symbol instead.</value>
```

Comment:
It would be nice to do a "first chance" lookup to see if `field` is a member of `this` (assuming there is a `this` parameter in scope), and in that case recommend user to use `this.field` instead.

Reply:
Let's wait on this. I'd prefer to use one warning rather than two but the wording may get confusing if we recommend something like "... or use `this.field` if a member of the instance" in cases where `field` is not an instance member.